src/Adapter/MSTest.TestAdapter/Execution/UnitTestRunner.cs

```diff
@@ -300,7 +300,7 @@ private class ClassCleanupManager
                         g => new HashSet<string>(g.Select(t => t.TestMethod.UniqueName))));
             _lifecycleFromMsTest = lifecycleFromMsTest;
             _lifecycleFromAssembly = lifecycleFromAssembly;
-            _reflectHelper = reflectHelper ?? new ReflectHelper();
```

Comment:
The code above already does not accept null, and we don't want to duplicate our cache.